src/java.desktop/windows/classes/sun/awt/windows/WFramePeer.java

```diff
@@ -80,11 +78,8 @@ public void toFront() {
     private native void setMaximizedBounds(int x, int y, int w, int h);
     private native void clearMaximizedBounds();
 
-    @SuppressWarnings("removal")
     private static final boolean keepOnMinimize = "true".equals(
```

Comment:
Can be replaced with `Boolean.getBoolean("sun.awt.keepWorkingSetOnMinimize");`.

It would be consistent with other updates, too.